### ISSUE TITLE
DMs to bot cause promise rejection

### ISSUE THREAD

**Issue by pianiststickman** [pinano-discord/Pinano-Discord-Bot#19]
Also from the most recent deployment:

```
(node:28564) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'id' of null
    at Client.client.on (/home/pinanostaff/pinano/library/client_events.js:31:63)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:28564) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 12)
```

Line 31 is the following:

```if (!client.settings.pinano_guilds.includes(message.guild.id))```

So we have a guildless message?

**Comment by sit** [comment]
Ah, these are DMs. If you try to DM your bot, it'll ignore your messages because of this.

But also, it appears that commands like `stats` also assume that your message has a guild. Perhaps we need the concept of a default guild or a way to specify the guild you are DM'ing for stats for.

**Comment by pianiststickman** [comment]
Fixed in #83
